### PR TITLE
fix: complex tag crash

### PR DESCRIPTION
**Pull request by npdev453**
Closes: #23

### CODE REVIEW

File: src/rules/noUnsafeQuery.ts [hunk 1]
```diff
@@ -34,14 +34,12 @@ const create = (context) => {
...
 
-      const { tag } = node.parent;
-
       const tagName =
         node.parent.tag?.name ??
         node.parent.tag?.object?.name ??
         node.parent.tag?.callee?.object?.name;
 
-      const legacyTagName = tag?.name.toLowerCase();
+      const legacyTagName = node.parent.name?.toLowerCase();
```
Comment (andymakingthings) [inline src/rules/noUnsafeQuery.ts:42]: This should possibly be:

```ts
const legacyTagName = node.parent.tag?.name?.toLowerCase();
```

Comment (gajus) [inline src/rules/noUnsafeQuery.ts:42]:
```suggestion
      const legacyTagName = node.parent.tag?.name?.toLowerCase();
```

Comment (andymakingthings) [reply src/rules/noUnsafeQuery.ts:42]: Did this get missed in the merge?

Comment (gajus) [reply src/rules/noUnsafeQuery.ts:42]: Added

Comment (npdev453) [reply src/rules/noUnsafeQuery.ts:42]: @andymakingthings I as remember there are shouldt be `tag` in parent because it in this case parent is not a tag

Comment (npdev453) [reply src/rules/noUnsafeQuery.ts:42]: also provided test should prove that it works now

Comment (andymakingthings) [reply src/rules/noUnsafeQuery.ts:42]: I've just confirmed it's no longer giving me an error with the latest code. Thanks!

**Pull request merged**